app/views/projects/_form.html.erb

```diff
@@ -54,6 +54,14 @@
       </div>
     </div>
 
+    <div class="form-group">
+      <div class="col-lg-offset-2 col-lg-4">
+        <%= form.check_box :deploy_to_new_pods %>
+        <%= form.label :deploy_to_new_pods, 'Deploy to New Pods'%>
+        <span class="help-block">This a pod-specific service, and should be deployed to each pod, including new pods.</span>
```

Comment:
samson does not know about "pods" ... as long as changing the deploy groups makes the stage copy-able then that should be enough for us ... and that will only work when it uses $DEPLOY_GROUPS ... and so far I have only seen that being used directly via cap `$DEPLOY_GROUPS` or kubernetes ... a project that does not use deploy-groups in the usual way will not be copy-able and will afaik not use $DEPLOY_GROUPS either